requirements.txt

```diff
@@ -1 +1 @@
-aenum==2.0.8
+aenum==2.1.2
```

Comment:
have to ask if we really need to use this enum still? That's maybe something I can look at in the future